scripts/install_wxpython.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if [[ -v TRAVIS ]]; then
+    pip install https://extras.wxpython.org/wxPython4/extras/linux/gtk2/ubuntu-14.04/wxPython-4.0.3-cp36-cp36m-linux_x86_64.whl
+fi
```

Comment:
This script grabs a prebuilt binary wheel of wxPython if TRAVIS is defined in the environment.  We'd need to update the path if we used a different Travis image sometime.

I plan on making this script robust enough to work on local dev machines as the single way to install wxPython for this project.